openapi/azure.openai.chat/openapi.yaml

```diff
@@ -0,0 +1,259 @@
+openapi: 3.0.0
+info:
+  x-ballerina-display:
+      label: Azure OpenAI Chat
+      iconPath: "icon.png"
+  title: Azure OpenAI Service API
+  description: >
+    This is a generated connector from [Azure OpenAI Chat Completions API v2023-03-15-preview](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#chat-completions/) OpenAPI specification.
+
+    The Azure Azure OpenAI Service REST API Chat Completions Endpoint will create completions for chat messages with the ChatGPT (preview) and GPT-4 (preview) models.
+  x-ballerina-init-description: >
+    The connector initialization requires setting the API credentials.
+
+    Create an [Azure](https://azure.microsoft.com/en-us/features/azure-portal/) account, an [Azure OpenAI resource with a model deployed](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource) and refer [this guide](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#authentication) to learn how to generate and use tokens
+  version: 2023-03-15-preview
+servers:
+  - url: https://{endpoint}/openai
+    variables:
+      endpoint:
+        default: your-resource-name.openai.azure.com
+security:
+  - bearer:
+      - api.read
+  - apiKey: []
+paths:
+  /deployments/{deployment-id}/chat/completions:
+    post:
```

Comment:
Add `x-ballerina-display` to all endpoints. Refer
https://github.com/hevayo/ballerina-openai-client/blob/fixes/client_openapi.yaml#L22-L23